manifest.yml

```diff
@@ -1,9 +1,11 @@
 ---
 applications:
-    - name: opre-ops-test
-      buildpacks:
-        - python_buildpack
-
+  - name: opre-ops-test
+    buildpack: python_buildpack
+    # TODO: Django docs say not to use the `runserver` command in prod;
+    # switch over to a production-ready webserver like gunicorn
+    command: cd opre_ops && python manage.py migrate && python manage.py runserver 0.0.0.0:8080
```

Comment:
Updated this infrastructure as code configuration to run our Django migrations before running the application server.

Reply:
Thanks for maintaining the note about not using `runserver` in prod!